desktop/apps/consignments2/client/actions.js

```diff
@@ -14,12 +35,42 @@ export function createSubmission () {
   }
 }
 
+export function fetchArtistSuggestions (value) {
+  return async (dispatch, getState) => {
+    try {
+      request.get(`${sd.API_URL}/api/v1/match/artists`)
+        .query({ visible_to_public: 'true', term: value })
+        .set('X-ACCESS-TOKEN', sd.CURRENT_USER.accessToken)
+        .end((err, res) => {
+          if (!err || res.ok) {
+            dispatch(updateArtistSuggestions(res.body))
+            dispatch(hideNotConsigningMessage())
+          }
+        })
+    } catch (error) {
+      console.error('error!', error)
```

Comment:
Server-side, we should be doing `catch(next)` to forward it to the error handling middleware (and eventual Sentry report).

Client-side, we should probably have something similar: at least the Sentry reporting. I also think `console.error` is fine even in prod b/c it will help debugging if an error occurs! Why would you want to hide it?